tests/helper_functions.py

```diff
@@ -395,3 +395,10 @@ def _is_importable(module_name):
         return True
     except ImportError:
         return False
+
+
+def disable_prevent_infer_pip_requirements_fallback_if(condition):
+    def decorator(f):
+        return pytest.mark.disable_prevent_infer_pip_requirements_fallback(f) if condition else f
```

Comment:
```suggestion
        return pytest.mark.allow_infer_pip_requirements_fallback(f) if condition else f
```

`allow_infer_pip_requirements_fallback` may be more straightforward.

Reply:
Yep! Much easier to understand without the double negative.

Reply:
I'll address this in a follow-up :)